apps/src/StudioApp.js

```diff
@@ -995,8 +1004,15 @@ StudioApp.prototype.onMouseMoveVizResizeBar = function (event) {
   visualizationColumn.style.maxWidth = (newVizWidth + vizSideBorderWidth) + 'px';
   visualization.style.maxWidth = newVizWidthString;
   visualization.style.maxHeight = newVizHeightString;
-  applyTransformScaleToChildren(visualization,
-      'scale(' + (newVizWidth / this.nativeVizWidth) + ')');
+
+  // We don't get the benefits of our responsive styling, so set height
+  // explicitly
+  if (!utils.browserSupportsCssMedia()) {
+    visualization.style.height = newVizHeightString;
```

Comment:
The short answer is that it's because height is what was broken, whereas i wasn't seeing any problems with width. I'll do a little more due diligence and see if it makes sense to be setting width too.